server/index.ts

```diff
@@ -0,0 +1,59 @@
+import createLnRpc, { LnRpcClientConfig } from "@radar/lnrpc"
+import bodyParser from "body-parser"
+import dotenv from "dotenv"
+import express from "express"
+import morgan from "morgan"
+import { homedir } from "os"
+import { join } from "path"
+
+import { IslandCurator } from "./src/curator"
+import { PaymentHandler } from "./src/payment"
+import { LevelDBFlashStore } from "./src/store"
+
+
+async function setupPaymentHandler(): Promise<PaymentHandler> {
+    const lnRpcHost = process.env.LNRPC_REMOTE_HOST
+    let config: LnRpcClientConfig
+    if (!lnRpcHost) {
+        const lnRpcNetwork = process.env.LNRPC_NETWORK || "testnet"
+        config = {
+            server: "localhost:10009",
+            macaroonPath: join(homedir(), `/.lnd/data/chain/bitcoin/${lnRpcNetwork}/admin.macaroon`),
+        }
+    } else {
+        config = {
+            server: `${lnRpcHost}:10009`,
+            tls: `./secret/${lnRpcHost}/tls.cert`,
+            macaroonPath: `./secret/${lnRpcHost}/admin.macaroon`,
+        }
+    }
+    console.log(`Connecting to LN RPC with config: ${JSON.stringify(config, null, 2)}`)
+    const lnRpc = await createLnRpc(config)
+    return new PaymentHandler(lnRpc)
+}
+
+async function run(listenPort: number): Promise<void> {
+    const db = new LevelDBFlashStore(__dirname + "/data/islandsdb")
+
+    const payments = await setupPaymentHandler()
+
+    const app = express()
+
+    app.use(bodyParser())
+
+    app.get("/test", (req, res) => res.end("OK"))
+
+    app.use("/island/genesis",
+        new IslandCurator("genesis", db, payments).createRouter())
```

Comment:
@geralddejong Got a better name for the "genesis" island?

Reply:
I expect we will have a number of islands.  Can it be "/island/{island-name}" ?  the client can decide which island to load.

Reply:
Right, but there is a hardcoded set of islands is there not?

Reply:
We could start with just one hardcoded, but eventually it'd be good to be able to choose which island to visit.